Objects/EventItem/Chicken.gd

```diff
@@ -0,0 +1,13 @@
+extends "res://Objects/EventItem/EventItem.gd"
+
+var chicken_speed: float = -100
+
+func _ready():
+	set_start_position(1100,575)
+	move_speed = chicken_speed
+
+func _process(delta):
+	move(delta)
```

Comment:
This line is unneaded, because in `EventItem.gd` `move` function is called every frame. So `_process` function is unneaded as well

Reply:
You are right